cpp/order/Steinhardt.cc

```diff
@@ -285,7 +285,7 @@ void Steinhardt::computeAve(const box::Box& box, const locality::NeighborList* n
                 m_QlmiAve.get()[index] /= neighborcount;
                 m_Qlm_local.local()[k] += m_QlmiAve.get()[index] / float(m_Np);
                 // Add the norm, which is the complex squared magnitude
-                m_QliAve.get()[i] += norm(m_Qlmi.get()[index]);
+                m_QliAve.get()[i] += norm(m_QlmiAve.get()[index]);
```

Comment:
This is the line that resolves #344. See that issue for details.

Reply:
Are there potentially more areas where the same error might have been made?

Reply:
I looked back through the current code and the PR that introduced this error. I did not find any other copy-paste errors. The current code has been massively simplified, so there are fewer places such a mistake could hide.